static/routes.config.php

```diff
@@ -31,18 +31,20 @@
 use Friendica\Module;
 
 $profileRoutes = [
-	''                                         => [Module\Profile\Index::class,             [R::GET]],
-	'/attachment/upload'                       => [Module\Profile\Attachment\Upload::class, [        R::POST]],
-	'/contacts/common'                         => [Module\Profile\Common::class,            [R::GET]],
-	'/contacts[/{type}]'                       => [Module\Profile\Contacts::class,          [R::GET]],
-	'/media'                                   => [Module\Profile\Media::class,             [R::GET]],
-	'/photos'                                  => [Module\Profile\Photos\Index::class,      [R::GET         ]],
-	'/photos/upload'                           => [Module\Profile\Photos\Upload::class,     [        R::POST]],
-	'/profile'                                 => [Module\Profile\Profile::class,           [R::GET]],
-	'/remote_follow'                           => [Module\Profile\RemoteFollow::class,      [R::GET, R::POST]],
-	'/schedule'                                => [Module\Profile\Schedule::class,          [R::GET, R::POST]],
-	'/status[/{category}[/{date1}[/{date2}]]]' => [Module\Profile\Status::class,            [R::GET]],
-	'/unkmail'                                 => [Module\Profile\UnkMail::class,           [R::GET, R::POST]],
+	''                                         => [Module\Profile\Index::class,              [R::GET]],
+	'/attachment/upload'                       => [Module\Profile\Attachment\Upload::class,  [        R::POST]],
+	'/attachment/browser'                      => [Module\Profile\Attachment\Browser::class, [R::GET]],
```

Comment:
Not sure why you added this route in the profile routes, unless we can use it to browse other people's attachments? I probably made the first mistake with the `attachment/upload` and `photos/upload` routes.

Reply:
I thought it was intended, so I adapted it the same way :D

So if you see no use case, I'd love to move Attachment/Photos to the `User` Module namespace and remove the `nickname` route-parameter, since I really don't see any usage for both upload and browser

Reply:
It's fine with me and it will simplify stuff in this PR.

Reply:
hmm .. Unfortunately, I think it was done with a reason:

https://github.com/friendica/friendica/blob/develop/src/Module/Profile/Photos/Upload.php#L79-L83

Reply:
Yes, it was meant for forum users to post media as the forum user, this can be scrapped as well (the `$visitor` variable).

Reply:
puh .. allright :D